setup.py

```diff
@@ -1,12 +1,18 @@
 # Import required functions
 from setuptools import setup, find_packages
+from pathlib import Path
+
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
```

Comment:
Are there plans to update the Readme to make this info more appropriate for installing from PyPI?

Reply:
Good shout.

I've also noticed there is an encoding issue with this line of code:
![image](https://user-images.githubusercontent.com/976274/199015020-03d9b18c-ba99-4512-b349-febe67960c42.png)